common/scala/src/main/scala/whisk/core/containerpool/logging/DockerToActivationFileLogStore.scala

```diff
@@ -75,21 +76,23 @@ class DockerToActivationFileLogStore(system: ActorSystem, destinationDirectory:
   protected val writeToFile: Sink[ByteString, _] = MergeHub
     .source[ByteString]
     .batchWeighted(bufferSize.toBytes, _.length, identity)(_ ++ _)
-    .to(LogRotatorSink(() => {
-      val maxSize = bufferSize.toBytes
-      var bytesRead = maxSize
-      element =>
-        {
-          val size = element.size
-          if (bytesRead + size > maxSize) {
-            bytesRead = size
-            Some(destinationDirectory.resolve(s"userlogs-${Instant.now.toEpochMilli}.log"))
-          } else {
-            bytesRead += size
-            None
+    .to(RestartSink.withBackoff(1.seconds, 60.seconds, 0.2) { () =>
```

Comment:
Can we name those parameters? It's a little hard to guess what they mean by just looking at it.

```scala
RestartSink.withBackoff(
  minBackoff = 1.seconds, 
  maxBackoff = 60.seconds, 
  randomFactor = 0.2)
```

Makes it a bit easier on the reader.